### ISSUE TITLE
Allow for Repo to be checked out directly onto a WordPress.org sandbox

### ISSUE THREAD

**Issue by dd32** [WordPress/wporg-mu-plugins#141]
In order to make ongoing maintenance easier, it would be much simpler if this Repo could be checked out directly onto a sandbox in the correct place.

However, as the `mu-plugins` folder exists within this repo, only files contained in that subdirectory are able to be put in place.

It would be ideal if this could be done via something similar to `git clone https://github.com/WordPress/wporg-mu-plugins wp-content/mu-plugins/wporg-mu-plugins` with a loader at `wp-content/mu-plugins/loader.php` stored outside of the repo which ultimately included a loader from #137

tl;dr: I'd like to remove the nested `mu-plugins` folder, and instead have the plugin folders sit alongside the build tools, to form a simpler development flow.
